hack/support-bundle-collector.sh

```diff
@@ -15,16 +15,20 @@ BUNDLE_DIR="${OUTPUT_DIR}/${NODE_NAME}"
 
 mkdir -p ${BUNDLE_DIR}
 
-OS_ID=$(bash -c "source $HOST_PATH/etc/os-release && echo \$ID")
-if [ -z "$OS_ID" ]; then
-    echo "Unable to determine OS ID"
-    exit 1
-fi
-
 if [ -n "$SUPPORT_BUNDLE_COLLECTOR" ]; then
     OS_COLLECTOR="collector-$SUPPORT_BUNDLE_COLLECTOR"
 else
-    OS_COLLECTOR="collector-$OS_ID"
+    OS_RELEASE_PATH=${HOST_PATH}/etc/os-release
+
+    if [ -f ${OS_RELEASE_PATH} ]; then
+        OS_ID=$(bash -c "source ${OS_RELEASE_PATH} && echo \$ID")
+        OS_COLLECTOR="collector-$OS_ID"
+
+        if [ -z "$OS_ID" ]; then
+            echo "Unable to determine OS ID"
+            exit 1            
+        fi
+    fi
```

Comment:
I just noticed this change would cause empty OS_COLLECTOR if OS_RELEASE_PATH does not exist, even though the current implementation does error handling afterward. 

Please just simply change to below, also avoid nest if condition as well.

```bash
if [ -n "$SUPPORT_BUNDLE_COLLECTOR" ]; then
    OS_COLLECTOR="collector-$SUPPORT_BUNDLE_COLLECTOR"
else
    OS_ID=$(bash -c "source ${HOST_PATH}/etc/os-release && echo \$ID")
    if [ -z "$OS_ID" ]; then
        echo "Unable to determine OS ID"
        exit 1
    fi
    
    OS_COLLECTOR="collector-$OS_ID"
fi
```

Reply:
Understood, I have re-updated it now. Thank you.